### PR TITLE
Remove markdownsafe, match exactly dependencies from setup.py

### PR DESCRIPTION
**Pull request by kinow**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by kinow** [comment]
Testing the metapackage locally with this version. Just got an exception running the `five` workflow.

image

So will leave as draft until I have managed to get the whole system working, avoiding having to release multiple times this conda package.

**Comment by kinow** [comment]
#TodayILearned

The ranges from SetupTools like `==1.*` do not work in Conda.

```
WARNING conda.models.version:get_matcher(528): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.*, but conda is ignoring the .* and treating it as 3
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2
WARNING conda.models.version:get_matcher(528): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2
```

They are ignored and replaced by nothing, which could be dangerous. So used the `>=1,<2` approach.

**Comment by kinow** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,24 +20,24 @@ requirements:
     - python
   run:
     - ansimarkup >=1.0.0
-    - click >=7.0
     - colorama >=0.4
```
Comment (oliver-sanders) [inline recipe/meta.yaml:23]: In cylc-flow this is `0.4.*` so `>=0.4` could cause install issues.

We really don't need to pin colorama to `0.4.*` so it's an easy fix for 8.0a3 but for now I guess we need to specify `colorama>=0.4,<=0.5`?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -20,24 +20,24 @@ requirements:
...
     - cryptography
-    - graphene ==2.1.6
-    - jinja2 >=2.10.1
-    - markupsafe >=1.1
-    - metomi-isodatetime ==1!2.0.2
+    - graphene >=2.1,<3
+    - jinja2 ==2.11.0,<2.12
+    - metomi-isodatetime ==1!2.0.1
     - protobuf ==3.12.1
     - pyasn1
```
Comment (oliver-sanders) [inline recipe/meta.yaml:30]: How did this get in here?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -20,24 +20,24 @@ requirements:
...
     - python
+    - pyzmq >=18.1.0,<19
```
Comment (oliver-sanders) [inline recipe/meta.yaml:32]: cylc-flow has `pyzmq==18.1.*` so I think we need:

```suggestion
    - pyzmq >=18.1.0,<18.2.0
```

File: recipe/meta.yaml (hunk 1 above)
Comment (kinow) [reply recipe/meta.yaml:23]: Oh, you are right! Let me fix that.

File: recipe/meta.yaml (hunk 2 above)
Comment (kinow) [reply recipe/meta.yaml:30]: That was from one of the very first builds, when alpha1 was ready to be released. I remember adding this `pyasn1` and `cryptography` when building the first recipe.

But cannot say whether it is still necessary. I can drop it and run some tests tomorrow to confirm whether this, and `cryptography` are necessary, and add a note here... though I don't know if it was some subcommand that failed, or if it was running a workflow, ...

Comment (oliver-sanders) [reply recipe/meta.yaml:30]: Ok, not urgent, would be good to stick a comment in if it turns out to be necessary (I've never heard of pyasn1 before).

Comment (kinow) [reply recipe/meta.yaml:30]: Had to google to see what was pyasn1 too :smile: I only remember adding it :-( created #8 to track that later.

**Pull request merged**